mapstory/importers.py

```diff
@@ -54,11 +54,13 @@ def import_file(self, *args, **kwargs):
 
         for layer in configuration_options:
             feature_type = layer.get('featureType')
-            datastore = feature_type['store']['name']
+            store_name = feature_type['store']['name']
```

Comment:
:+1: cool, just making sure I didn't overlook a change coming in from importer itself or something like that